packages/components/tabs/src/TabsList.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { cva } from 'class-variance-authority'
+
+export const wrapperStyles = cva(['relative flex'])
```

Comment:
If I'm not wrong, this is to take care of custom styling, as you'll see in implementation:
```tsx
<div className={wrapperStyles({ className })} ref={wrapperRef}>...
```

Reply:
You could do:
```tsx
import { cx } from 'class-variance-authority'

<div className={cx(wrapperStyles(), className} ref={wrapperRef}>...
```

My point was that using `cva` might be a bit overkill in this case, since it's typically used to handle more complex styles. If you're only declaring a few static classes, it might be more appropriate to use a simpler solution 🤷

Reply:
Maybe I don't get your point, but here in your last proposal, I should use `tw()` and `cx()`, instead of simply `cva()`? Not sure which option is the most overkill here :sweat_smile:

Reply:
Actually, in your case (using only two classes), you could simply do:
```tsx
<div className={cx('relative flex', className} ref={wrapperRef}>...
```

I mean, you don't really need to extract styles when it's only a couple of classes,

Regarding the use of `cva` for static styles, it's more of a subjective matter.
Personally, I tend to use `cva` for dealing with complex styles that have variants. Nevertheless, if you prefer to use `cva` every time for consistency, that's alright with me 👍